main_test.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestwordOccurrences(t *testing.T) {
```

Comment:
Other useful tests:
 * multiple occurrences of the same word
 * a non-space separator